README.md

```diff
@@ -28,11 +32,14 @@ Setup the application:
 
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
+If `gearbox setup-app` fails, try specifying `pip3` in the previous two steps.
```

Comment:
This is really system dependent... on some systems you'll even need something like ``python3.6 -m pip``. It's not really the job of our README to tell people how to use their system, and therefore this should not be included.